ingeniamotion/configuration.py

```diff
@@ -708,15 +708,13 @@ def is_sto1_active(self, servo: str = DEFAULT_SERVO, axis: int = DEFAULT_AXIS) -
             axis : servo axis. ``1`` by default.
 
         Returns:
-            return value of STO1 bit.
+            returns True when the STO 1 input is active (bit is 0)
+            returns False when the STO 1 input is inactive (bit is 1)
```

Comment:
Be careful with docstring render in the docs:
![image](https://github.com/user-attachments/assets/14540fc3-fd4d-44e3-b9e7-bc6187e73b16)